cli/src/commands/log.rs

```diff
@@ -47,18 +47,21 @@ use crate::ui::Ui;
 /// Renders a graphical view of the project's history, ordered with children
 /// before parents. By default, the output only includes mutable revisions,
 /// along with some additional revisions for context. Use `jj log -r ::` to see
-/// all revisions. See `jj help -k revsets` (or
-/// <https://jj-vcs.github.io/jj/latest/revsets/>) for information about the
-/// syntax.
+/// all revisions. See [`jj help -k revsets`] for information about the syntax.
```

Comment:
Let's put the URL right under the paragraph. Otherwise, it's too far.

Another option I considered would be to put it at the end of the list of footnotes, so that it doesn't interfere with users looking for `[Immutable revisions]`, but that seems more arbitrary.

I'd also be OK with reverting this one link (and adding `<>`), but that also feels arbitrary.

Reply:
moved it to the paragraph.